echo-core/src/test/groovy/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgentSpec.groovy

```diff
@@ -92,6 +92,45 @@ class AbstractEventNotificationAgentSpec extends Specification {
     // notifications ON, stage is synthetic
     fakeStageEvent("orca:stage:complete", "stage.complete", false, true)                      || 0
   }
+  
+  @Unroll
```

Comment:
MIGHT add a comment here...
```
/* 
Orca sends two events on manual judgements unfortunately.  one is a sage started, but then a DIFFERENT 
notification explicitly for manual judgements.  We want to only fire for manual judgement custom 
notification types here.
*/
```
JUST so someone else hits this won't get bit by it.